.github/workflows/build.yml

```
@@ -0,0 +1,61 @@
+        with:
+          distribution: 'temurin'        # ← this is now required
+          java-version: '21'
+          java-package: 'jdk'
+          check-latest: false
+          server-id: 'github'            # for generated Maven settings.xml
+          server-username: '${{ github.actor }}'
+          server-password: '${{ secrets.GITHUB_TOKEN }}'
+          overwrite-settings: true
+          job-status: 'success'
+          token: '${{ secrets.GITHUB_TOKEN }}'
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)